src/Servers/Kestrel/Core/src/Internal/Http/HttpMethod.cs

```diff
@@ -3,19 +3,75 @@
 
 namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http;
 
+/// <summary>
+/// This API supports framework infrastructure and is not intended to be used
+/// directly from application code.
+/// </summary>
 public enum HttpMethod : byte
 {
+    /// <summary>
+    /// This API supports framework infrastructure and is not intended to be used
+    /// directly from application code.
+    /// </summary>
```

Comment:
💡 Consider using `<inheritdoc />` for this

```suggestion
    /// <inheritdoc cref="HttpMethod" />
```

Reply:
You can also create a type specifically for this purpose, and then link all documentation to it for elements to have consistent wording.

Reply:
I'll do this the next time around. It's more work now to update, but thanks for the suggestion 👍🏽